### PR TITLE
Universal login pages

### PR DESCRIPTION
**Pull request by gestchild**
Updates the universal login template to look like [new designs](https://app.zeplin.io/project/5aab926b4b6efde01259e959/dashboard?tag=Requesting%20-%20member%20sign%20in)

References #6805

### CODE REVIEW

File: identity/liquid-templates/template.html [hunk 1]
```diff
@@ -64,81 +64,63 @@
...
+        text-align: left!important;
+      }
+      ._prompt-box-outer button {
+        width: auto!important;
+      }
+      ._prompt-box-outer div {
+        padding-left: 0!important;
+        min-height: auto!important;
+      }
+      ._prompt-box-outer .success-email {
```
Comment (jamesgorrie) [inline identity/liquid-templates/template.html:77]: FMI - are these auth0 styles?

File: identity/liquid-templates/template.html [hunk 2]
```diff
@@ -64,81 +64,63 @@
...
-        color: #121212;
         line-height: 1.5;
       }
-      .background {
-          background: #ffffff;
-        }
-      @media (min-width: 920px) {
-        .background {
-          background: linear-gradient(90deg, #e8e8e8 50%, #ffffff 50%);
+      @media (min-width: 803px) {
```
Comment (jamesgorrie) [inline identity/liquid-templates/template.html:89]: Why 803?

File: identity/liquid-templates/template.html [hunk 3]
```diff
@@ -64,81 +64,63 @@
...
-      }
-      @media (min-width: 920px) {
-        .container {
-          flex-direction: row-reverse;
-        }
+      .header {
+        background: #ffffff;
+        border-bottom: 1px solid #d9d6ce;
+        height: 85px;
+        padding-top: 23px;
```
Comment (jamesgorrie) [inline identity/liquid-templates/template.html:98]: Again, some funny values?

File: identity/liquid-templates/template.html [hunk 4]
```diff
@@ -150,82 +132,115 @@
...
+        margin: 0 0 12px 0;
+      }
+      p {
+        margin: 0 0 22px 0;
       }
       body a {
         font-size: 16px;
         color: #121212;
         text-decoration: underline;
+        font-weight: normal!important;
```
Comment (jamesgorrie) [inline identity/liquid-templates/template.html:194]:
```suggestion
        font-weight: normal !important;
```

**Review by jamesgorrie — APPROVED** [review]
HOw important do you think it is that we get this app into nextjs?

File: identity/liquid-templates/template.html (hunk 1 above)
Comment (gestchild) [reply identity/liquid-templates/template.html]: yep

File: identity/liquid-templates/template.html (hunk 3 above)
Comment (gestchild) [reply identity/liquid-templates/template.html:100]: This places the logo in the same position as the main site

**Pull request merged**